### DISCUSSION TITLE
Enum.HasAnyFlag

### DISCUSSION THREAD

**Discussion by ulfemsoy** [dotnet/winforms#12129] [Ideas] [answered]
The current Enum.HasFlag(value, flag) is defined as:
(value & flag) == flag

I suggest to introduce another common method called Enum.**HasAnyFlag**(value, flag)  defined as
(value & flag) **!= 0**

**Comment by elachlan** [comment, marked as answer]
I don't think this is related to winforms and is more runtime? You would have to raise an API suggestion there.

**Reply by ulfemsoy** [reply]
@elachlan, good point. I have registered the new suggestion here:
https://github.com/dotnet/runtime/discussions/107824
